### PR TITLE
BZ-1724488: Added description of openshift_docker_ent_reg variable

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1724488/install/configuring_inventory_file.html#configuring-cluster-variables

For BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1724488

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@vrutkovs Can you review the description I added for `openshift_docker_ent_reg` here [1]? I added a sentence about setting it to an empty string, that I found in the related BZs. Wanted to make sure this was all accurate. Thanks!

[1] http://file.rdu.redhat.com/~ahoffer/2019/BZ-1724488/install/configuring_inventory_file.html#configuring-cluster-variables

**Review by vrutkovs — APPROVED** [review]

**Comment by bergerhoffer** [comment]
@openshift/team-documentation For peer review please.

File: install/configuring_inventory_file.adoc [hunk 1]
```diff
@@ -313,6 +313,9 @@ of the host. `0.0.0.0/0` is not a valid setting for the IP address.
 *docker* configuration. Block the listed registries. Setting this to `all`
 blocks everything not in the other variables.
 
+|`openshift_docker_ent_reg`
+|An additional registry that is trusted by the container runtime, when `openshift_deployment_type` is set to `openshift-enterprise`. If you set `openshift_docker_ent_reg=''`, then `registry.access.redhat.com` will not be added to the *docker* configuration.
```
Comment (kalexand-rh) [inline install/configuring_inventory_file.adoc]: @bmcelvee, does this note make sense for 3.11?

**Review by kalexand-rh — COMMENTED** [review]
The change LGTM, but I'd be happier if Brandi took a look. I know that the default registry changed, and I'm not sure if mentioning the old one is right here.

File: install/configuring_inventory_file.adoc (hunk 1 above)
Comment (bmcelvee) [reply install/configuring_inventory_file.adoc]: I think it's fine for now. We still don't have a clear timeline for the sunsetting of `registry.access.redhat.com`. If you want, you can replace it with `registry.redhat.io` or maybe "then `registry.access.redhat.com`and `registry.redhat.io` will not..."

Comment (bergerhoffer) [reply install/configuring_inventory_file.adoc]: Thanks @bmcelvee!

@vrutkovs Does it make sense here to say that both`registry.access.redhat.com` and `registry.redhat.io` will not be added?

Comment (vrutkovs) [reply install/configuring_inventory_file.adoc]: @bergerhoffer sounds good

Comment (wzheng1) [reply install/configuring_inventory_file.adoc]: In 3.11, `openshift_docker_ent_reg` default to be `registry.redhat.io` , in 3.10 `openshift_docker_ent_reg` default to be `registry.access.redhat.com`, It maybe better to separate these two?

**Comment by bergerhoffer** [comment]
Merging as this was verified by QE.

**Pull request merged**